database/factories/ModelFactory.php

```diff
@@ -16,9 +16,86 @@
     static $password;
 
     return [
-        'name' => $faker->name,
-        'email' => $faker->unique()->safeEmail,
-        'password' => $password ?: $password = bcrypt('secret'),
-        'remember_token' => str_random(10),
+        'username' => $faker->firstName(),
+		'email' => $faker->unique()->userName().'@gmail.com',
+		'password' => bcrypt('123456'),
+		'fullname' => $faker->firstName().' '.$faker->lastName(),
+		'created_at'=> new DateTime(),
```

Comment:
no need these. You already define `$timestamp = true` in the model, then `created_at` and `updated_at` are auto filled